__tests__/app.test.js

```diff
@@ -38,3 +39,68 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api").expect(200);
+  });
+  test("responds with an object", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        expect(typeof response).toBe("object");
+      });
+  });
+  test("responds with an object containing a property for each request", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        const { endpoints } = response.body;
+        expect(endpoints.hasOwnProperty("GET /api")).toBe(true);
+        expect(endpoints.hasOwnProperty("GET /api/topics")).toBe(true);
+        expect(endpoints.hasOwnProperty("GET /api/articles")).toBe(true);
+      });
+  });
+  test("responds with a correct object containing the endpoint objects", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        const { endpoints } = response.body;
+        const expected = {
+          "GET /api": {
```

Comment:
what will happen when you update the endpoints.json in the next endpoint? this test is very brittle, but could just be as simple as comparing the result of the get request to the json you've required in